src/core/build_target.go

```diff
@@ -28,6 +28,9 @@ const GenDir string = "plz-out/gen"
 // BinDir is the output directory for binary targets.
 const BinDir string = "plz-out/bin"
 
+// SubrepoDir is the output directory for targets that define subrepos.
+const SubrepoDir = "plz-out/sub"
```

Comment:
I wonder if `repos` is more descriptive?

Reply:
I would join both answers - `subrepos`